misc/fluentd/Dockerfile

```diff
@@ -0,0 +1,42 @@
+FROM alpine@sha256:58e1a1bb75db1b5a24a462dd5e2915277ea06438c3f105138f97eb53149673c4
```

Comment:
Could we use `fluentd/fluentd` as the base (`FROM` line) and still configure it ourselves?  Either that or just using the default format/logging of the `fluentd/fluentd` image would be fine I think.